CMakeLists.txt

```diff
@@ -60,10 +60,10 @@ OPTION(USE_HTTPS			"Enable HTTPS support. Can be set to a specific backend" ON)
 OPTION(USE_GSSAPI			"Link with libgssapi for SPNEGO auth"			OFF)
 OPTION(USE_STANDALONE_FUZZERS		"Enable standalone fuzzers (compatible with gcc)"	OFF)
 OPTION(VALGRIND				"Configure build for valgrind"				OFF)
-OPTION(USE_EXT_HTTP_PARSER		"Use system HTTP_Parser if available"			 ON)
 OPTION(DEBUG_POOL			"Enable debug pool allocator"				OFF)
 OPTION(ENABLE_WERROR			"Enable compilation with -Werror"			OFF)
 OPTION(USE_BUNDLED_ZLIB    		"Use the bundled version of zlib"			OFF)
+OPTION(USE_BUNDLED_HTTP_PARSER  	"Use the bundled HTTP Parser"				 ON)
```

Comment:
Ah, didn't know we already had an option. If we're about to change, then we should use a generic `USE_HTTP_PARSER=(bundled|system)`, if you ask me. The reasoning is that this would be much easier to extend. For example at a future point, we'd be able to add an option `USE_HTTP_PARSER=/path/to/http_parser`.